src/User32/User32+DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains nested type <see cref="DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS"/>
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Describes per-monitor DPI scaling behavior overrides for child windows within dialogs. The values in this enumeration are bitfields and can be combined.
+        /// </summary>
+        /// <remarks>
+        /// This enum is used with SetDialogControlDpiChangeBehavior in order to override the default per-monitor DPI scaling behavior for a child window within a dialog.
+        ///
+        /// These settings only apply to individual controls within dialogs.The dialog-wide per-monitor DPI scaling behavior of a dialog is controlled by <see cref="DIALOG_DPI_CHANGE_BEHAVIORS"/>.
+        /// </remarks>
+        public enum DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS : int
```

Comment:
At least some uses of this enum treat this as a flags enum (where multiple values can be set). Can you add `[Flags]` here?